examples/ucp_hello_world.c

```diff
@@ -220,9 +227,14 @@ static int run_ucx_client(ucp_worker_h ucp_worker)
 
     /* Send client UCX address to server */
     ep_params.field_mask      = UCP_EP_PARAM_FIELD_REMOTE_ADDRESS |
-                                UCP_EP_PARAM_FIELD_ERR_HANDLING_MODE;
+                                UCP_EP_PARAM_FIELD_ERR_HANDLING_MODE |
```

Comment:
what do you think about asking for ERR handling if `-e*` is specified by the user?

Reply:
not get about ```asking```

Reply:
actually it is not needed: in case if ep_params.err_mode parameter is not set to handle errors then all other err-handling arguments are ignored

Reply:
yes, I see
but my idea was to not ask UCP for ERR_HANDLING to ensure that UCP could select the best devices (since SHM doesn't support ERR_HANDLING, the UCP hello world never selects SHM)

Reply:
ah I see, thank you